client/src/app/api/models.ts

```diff
@@ -658,3 +658,8 @@ export interface WaveWithStatus extends MigrationWave {
   fullApplications: Application[];
   allStakeholders: StakeholderWithRole[];
 }
+
+export type Fact = {
+  name: string;
+  data: any;
```

Comment:
I was going to suggest we replace `any` with `unknown` here to prevent any issues with code assuming it's a string (JSON.stringify won't care what it is), but since we're going to revisit this to support structured facts it's probably fine to leave it as-is (and it stands out as something to be fixed anyway which is maybe good)